docs/content/docs/projects/BuildChecks.md

```diff
@@ -240,3 +240,45 @@ buildChecks {
     } 
 }
 ```
+
+### Incremental KAPT
+
+This check verifies that all KAPT annotation processors are support incremental annotation processing 
+if it is [enabled](https://kotlinlang.org/docs/reference/kapt.html#incremental-annotation-processing-since-1330) (`kapt.incremental.apt=true`). 
+Because if one of them does not support it then whole incremental annotation processing won't work at all.
+
+Incremental KAPT check allows to set up one of three behavioral modes:
+- `"none"` -- check is disable
+- `"warning"` -- prints warning in build log (default behaviour)
+- `"fail"` -- fail whole build
+
+```groovy
+buildChecks {
+    incrementalKapt {
+        mode = "fail"
+    }
+}
+```
+
+Right now the check verifies the following processors:
```

Comment:
It is important info. We should move it upper after the motivation block.
We could simplify it to `Supported processors`

Reply:
I've changed the title and moved this block closer to the beginning of the section.